templates/internet-of-things/smart-displays.html

```diff
@@ -7,377 +7,541 @@
-          <a href="https://ubuntu.com/engage/pudo-case-study">Pudo brings convenience and sustainability to last-mile logistics with smart lockers software&nbsp;&rsaquo;</a>
-        </h3>
       </div>
     </div>
   </section>
 
-  <section class="p-strip is-bordered">
-    <div class="row">
-      <div class="col-8">
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
         <h2>Partners</h2>
+      </div>
+      <div class="col">
         <p>
           Get in touch with one of our partners to help you build your Digital Signage or Kiosk solution using Ubuntu as the foundation of your technology stack.
         </p>
       </div>
     </div>
-    <div class="row u-equal-height">
-      <div class="col-4 p-card">
-        <header class="no-border u-align--center u-vertically-center u-hide--small"
-                style="min-height: 7rem">
-          <a href="https://www.screenly.io/">
-            {{ image(url="https://assets.ubuntu.com/v1/be292257-Dark%20Logo.svg",
-                        alt="Screenly",
-                        width="200",
-                        height="46",
-                        hi_def=True,
-                        loading="lazy") | safe
-            }}
-          </a>
-        </header>
-        <hr class="p-rule--muted u-hide--small" />
-        <h3 class="p-card__title">Screenly</h3>
-        <p class="p-card__content">
-          Screenly provides the hardware and software you need to manage digital signs at scale. Update and schedule content, monitor screen health, and show content from your existing tech stack.
-        </p>
-        <a href="https://www.screenly.io/">Learn more about Screenly&nbsp;&rsaquo;</a>
-        <a href="/engage/screenly-empowers-security-conscious-enterprises">Read about the case study&nbsp;&rsaquo;</a>
-      </div>
-      <div class="col-4 p-card">
-        <header class="no-border u-align--center u-vertically-center u-hide--small"
-                style="min-height: 7rem">
-          <a href="https://broadsign.com/">
-            {{ image(url="https://assets.ubuntu.com/v1/56fec042-Broadsign_Logotype_rgb.svg",
-                        alt="Broadsign",
-                        height="76",
-                        width="250",
-                        hi_def=True,
-                        loading="lazy") | safe
-            }}
-          </a>
-        </header>
-        <hr class="p-rule--muted u-hide--small" />
-        <h3 class="p-card__title">Broadsign</h3>
-        <p class="p-card__content">
-          An end-to-end software solution built on Ubuntu Core for media owners and buyers to plan, manage and deliver smart displays of any kind..
-        </p>
-        <a href="https://broadsign.com/">Learn more about Broadsign&nbsp;&rsaquo;</a>
+    <div class="row">
+      <div class="col-start-large-7 col-6">
+        <div class="row--50-50">
+          <div class="p-equal-height-row p-equal-height-row-50-50">
```

Comment:
`<div class="row--50-50">` seems unnecessary as the div below is also specifying row-50-50 i.e., `<div class="p-equal-height-row p-equal-height-row-50-50">`